### PR TITLE
chore: Bump MPC on prod-m01

### PR DESCRIPTION
**Pull request by ifireball**
Using the (currently unused) m01 cluster as testing grounds for the new
MPC version against the older Tekton version used in PROD.

### CODE REVIEW

**Review by mshaposhnik — APPROVED** [review]

**Comment by ifireball** [comment]
@arewm , @hugares , @gbenhaim we need your approval because of the change to the application set files.

**Review by dirgim — APPROVED** [review]
/lgtm
/approve

File: argo-cd-apps/base/member/infra-deployments/multi-platform-controller/multi-platform-controller.yaml [hunk 1]
```diff
@@ -14,7 +14,9 @@ spec:
                 environment: staging
                 clusterDir: ""
           - list:
-              elements: []
+              elements:
+                - nameNormalized: stone-prod-m01
```
Comment (hugares) [inline argo-cd-apps/base/member/infra-deployments/multi-platform-controller/multi-platform-controller.yaml:18]: the name of the cluster is stone-prd-m01, you need to minimally change the nameNormalized to stone-prd-m01 but to stay consistent, if you change clusterDir, you also need to rename the folder in component

**Review by ifireball — COMMENTED** [review]
> the name of the cluster is stone-prd-m01, you need to minimally change the nameNormalized to stone-prd-m01 but to stay consistent, if you change clusterDir, you also need to rename the folder in component 

I will fix the cluster name in a new PR, not going to make other changes because I hope to remove this hack soon.

**Pull request merged**